### PR TITLE
Add jwt configuration interface

### PR DESCRIPTION
**Pull request by tzheleznyak**
### What does this PR do?
Adds JWTConfigurationInterface

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [x] The changes in this PR do not affect the Conjur API

### CODE REVIEW

File: app/domain/authentication/authn_jwt/jwt_configuration_interface.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Authentication
+  module AuthnJWT
+    class JWTConfigurationInterface
```
Comment (orenbm) [inline app/domain/authentication/authn_jwt/jwt_configuration_interface.rb:3]: can you please add a comment on this class?

**Review by nessiLahav — APPROVED** [review]
Just add comments as Oren requested

File: app/domain/authentication/authn_jwt/jwt_configuration_interface.rb (hunk 1 above)
Comment (tzheleznyak) [reply app/domain/authentication/authn_jwt/jwt_configuration_interface.rb:4]: Added comment

File: app/domain/authentication/authn_jwt/jwt_configuration_interface.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module Authentication
+  module AuthnJWT
+    # Interface containing JWT configuration functions to be implemented in JWTConfiguration class for a vendor
```
Comment (orenbm) [inline app/domain/authentication/authn_jwt/jwt_configuration_interface.rb:3]: i can't say that i understand what a `JWTConfiguration class for a vendor` will be but i'm ok with merging this and waiting for the full implementation to understand

**Review by orenbm — APPROVED** [review]

**Pull request merged**